services/parser-database/app.py

```diff
@@ -50,5 +50,5 @@ def get_article_by_number_in_memory(id):
 
 
 if __name__ == '__main__':
-    parse_all_documents()
+    #parse_all_documents()
```

Comment:
This shouldn't be commented should it?

Reply:
It is left out so that every-time the app starts it doesn't have to parse the doc and I can test quickly.